spring-cloud-commons/src/main/java/org/springframework/cloud/client/serviceregistry/Registration.java

```diff
@@ -0,0 +1,7 @@
+package org.springframework.cloud.client.serviceregistry;
+
+/**
+ * @author Spencer Gibb
+ */
+public interface Registration {
```

Comment:
Yeah, cause looking at the test I don't really see any benefit of having this `Registration` interface. Can you explain that?